### DISCUSSION TITLE
Wallet şifre değiştirme

### DISCUSSION THREAD

**Discussion by burakinalcrypto** [IBAX-io/go-ibax#769] [Soru Cevap - Türkçe]
Arkadaşlar merhaba,

Wallet şifresini nasıl değiştirebiliriz? Bilen var mı? Sitede böyle bir yer göremedim. Private key ve seed bütün bilgiler var. Hesabıma daha önceden walletı bağlamıştım ama şifre hatası alıyorum.

Teşekkürler

**Comment by Yusufknk** [comment]
Cüzdanın sistem gelişimi devam etmektedir. Bu dahil en tüm sorunlar en kısa zamanda giderilecektir.
